kymatio/scattering1d/frontend/base_frontend.py

```diff
@@ -558,9 +558,7 @@ def build(self):
         self.log2_F = math.floor(math.log2(self.F))
 
         # Compute the minimum support to pad (ideally)
-        phi_f = gauss_1d(N_input_fr, self.sigma0/max(self.F, 2 ** self.J_fr))
-        min_to_pad_fr = 3 * compute_temporal_support(
-            phi_f.reshape(1, -1), criterion_amplitude=1e-3)
+        min_to_pad_fr = 8 * min(self.F, 2 ** self.J_fr)
```

Comment:
For my own knowledge, is this 8 arbitrarily selected for now?

Reply:
good catch. yes, it's a completely made-up number which, in what i've tried so far, happens to pass the padding tests. but it should be revised and potentially optimized in a later PR. this is worthy of a separate discussion.

Reply:
great, just clarifying. i take it at this point we're padding every path to the same length?

Reply:
> we're padding every path to the same length?

yes. doing otherwise would require to make `filters_fr` not just multiresolution but also multi-support. out of scope for now.